scripts/generate-values/base-renku-values.yaml.template

```diff
@@ -21,12 +21,6 @@ gateway:
 graph:
   gitlab:
     url: {gitlab_url}
-  tokenRepository:
-    tokenEncryption:
-      secret: <use `openssl rand -hex 8 | base64`>
-  webhookService:
-    hookToken:
-      secret: <use `openssl rand -hex 8 | base64`>
```

Comment:
question: Why are these removed? They are not used at all? Because these are not the values you automated in this PR.